troopers.sh

```diff
@@ -1,76 +1,89 @@
 #!/bin/bash
 
-prefix="`dirname $0`/"          # prefix should be with "/" in the end
+trap "cleanup" EXIT
```

Comment:
This will trap any exit signal. and call the `cleanup` function. So you can interrupt the script at any time without having leftover cookie files. (It also gets triggered  when exiting normally)